public/plugin.tsx

```diff
@@ -157,8 +204,21 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     if (config.multitenancy.enabled) {
       addTenantToShareURL(core);
     }
+
+    if (config.multitenancy.enable_aggregation_view) {
+      const columns = deps.savedObjectsManagement.columns.getAll();
```

Comment:
this declaration is not useful as you a defining a constant inside if block which has not scope outside and cannot be used

Reply:
This IF block is not needed. removed. Thanks for pointing out!